src/Spd.Utilities.Dynamics/DynamicsContextLookupHelpers.cs

```diff
@@ -429,5 +429,16 @@ public static string LookupLicenceCategoryKey(Guid? licenceCategoryId)
                     throw;
             }
         }
+        public static async Task<contact?> GetContactByInfo(this DynamicsContext context, contact contact, CancellationToken ct)
+        {
+
+            var query = context.contacts.Where(a => a.firstname == contact.firstname &&

```

Comment:
if there is no driver licence number, we always create new contact.

Reply:
I changed it, but Im not sure because in the confluence It seem that if the driverlicense has value, check it, otherwise ignore it. (that's my understanding)